frontend/src/composables/useRequestSourceInputValidations.ts

```diff
@@ -41,12 +41,13 @@ export const useRequestSourceInputValidations = (options: {
       const selectedTokenDecimals = selectedTokenValue.decimals;
       const selectedAmountRules = {
         $autoDirty: true,
-        numeric,
         required: helpers.withMessage('Amount is required', required),
         isMatchingDecimals: helpers.withMessage(
           `Max. number of decimals: ${selectedTokenDecimals}`,
           makeMatchingDecimalsValidator(selectedTokenDecimals),
         ),
+        numeric: helpers.withMessage('Must be a positive number', numeric),
+        minValue: helpers.withMessage('Must be a positive number', minValue(0)),
```

Comment:
Reason behind using this message is:
I had an issue in displaying messages based on priority. 
For example, when the input is '-123', vuelidate's `numeric` validator will mark the value as invalid. Same goes for `minValue` validator.
I thought, changing to this message explains both cases pretty well to the user so i don't need to go the extra way of writing more custom validator functions and tests for them.